en/airframes_experimental/traxxas_stampede.md

```diff
@@ -0,0 +1,122 @@
+# Traxxas Stampede VXL
+
```

Comment:
Great to have a short paragraph here about why this vehicle selected and what the conversion to autopilot required - e.g. I assume because this is a nice big frame at a good price point. From your diagram all you needed to do was create a mount for the autopilot electronics.